conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,258 @@
+# This is CORTX GConf (Config Store) which serves as input 
+# to all the CORTX Services
+# Location: /etc/cortx/cluster.conf 
+
+cortx:
+  external:
+    consul:
+      admin: admin
+      endpoints:
+      - http://consul-server.cortx-cluster.com:80
+      secret: MG12NU5BVjBVeWpBSWJuSV9uVjV5UzUxN3lTSVNkNUk1YkRzbjd2QkE9PQ==
+    kafka:
+      admin: admin
+      endpoints:
+      - kafka-server1.cortx-cluster.com
+      - kafka-server2.cortx-cluster.com
+      secret: TmRPNThMSm5GOVl5RHgwTVFfaTRJZ3k3YUUwMldmNkdxRE1KYVIxc0E9PQ==
+      
+  common:
+    product_release: LC              # TBD: To be removed
+    security:
+      device_certificate: /etc/cortx/solution/ssl/stx.pem
+      domain_certificate: /etc/cortx/solution/ssl/stx.pem
+      ssl_certificate: /opt/seagate/cortx/s3/install/haproxy/ssl/s3.seagate.com.pem
+    service:
+      admin: admin
+      secret: Z0FBQUFBQmhVY1dCdzJ6RkRXT3FSVjJXTWtUM0paZlB1LXFRRW0xNy1rRGhJLVpHUjcxMkVOeDZ6
+        MG9KWmpsM21yTmprZTJkYTBHUU1OWVl4dnZSaVRTeEs4UEdReHhkR1E9PQ==
+    setup_size: small                # TBD: To be removed and derived from limits
+    setup_type: K8                   # To be removed
+    storage:
+      config: /etc/cortx
+      local: /etc/cortx
+      log: /share/var/log/cortx/
+      shared: /share                 # To be removed
+      
+  csm:
+    agent:
+      endpoints:
+      - https://csm.seagate.com:8081
+    email_address: cortx@seagate.com # Optional 
+    mgmt_admin: cortxadmin
+    mgmt_secret: UlU1U0NrYklraS0ta3BOamN5X2J1VmUwRnJXTUFlb2JDdkItY1B3ZEE9PQ==
+      
+  s3:
+    auth_server: auth_server
+    auth_admin: admin
+    auth_secret: TVRGNDBQVnY3OXJ5bG4tLTQwSjNZQ0xQZVJiczBmUVNqcXFvTXdvNHc9PQ==
+    data:
+      endpoints:
+      - http://s3.seagate.com:80
+      - https://s3.seagate.com:443
+    iam:
+      endpoints:                     # To be removed
+      - https://iam.seagate.com:9443
+    internal:
+      endpoints:
+      - http://s3.seagate.com:28049
+    io_max_units: 8
```

Comment:
> io_max_units

As per our last discussion, these are motr units which any client can use, including RGW. 
Further though, there was a discussion to suggest value for this can be dynamic based on environment type large, medium, small. 
@sachitanands @yeshpal-jain-seagate please confirm.

Reply:
had a discussion with Madhavrao yesterday. As of now we can go ahead with io_max_units as 8 . later we can change it.